syzygy/operations.py

```diff
@@ -324,15 +324,38 @@ class RenameField(StagedOperation, operations.RenameField):
     instances where a rename operation is safe to perform.
     """
 
+    # XXX: Explicitly define the signature as migration serializer rely on
+    # __init__ introspection to assign the kwargs.
+    def __init__(self, model_name, old_name, new_name, stage):
+        super().__init__(
+            model_name=model_name, old_name=old_name, new_name=new_name, stage=stage
+        )
+
```

Comment:
I was thinking of taking advantage of [PEP-0362](https://peps.python.org/pep-0362/#visualizing-callable-objects-signature) by implementing a `StagedOperation.__init_subclass__` that sets an `__init__.__signature__` but I figured this was less arcane.

Reply:
For reference here's what it would look like https://github.com/charettes/django-syzygy/commit/26ad4fbbd5c4878539d236a2b424edf612201deb.

Definitely not worth the complexity tradeoff IMO.